Apps/Playground/Shared/Playground/GraphicsApiInfo.cpp

```diff
@@ -0,0 +1,47 @@
+#include "GraphicsApiInfo.h"
+
+#define STRINGIZEX(x) #x
+#define STRINGIZE(x) STRINGIZEX(x)
+
+namespace Playground::Internal
+{
+    namespace
+    {
+        constexpr auto JS_CLASS_NAME = "GraphicsApiInfo";
+    }
+
+    class GraphicsApiInfo final : public Napi::ObjectWrap<GraphicsApiInfo>

```

Comment:
I exposed the graphics API's name as a static member of a class, but maybe it would be better to add it directly to `global` as something like `GraphicsApi_Name`?

Reply:
I would recommend making this a static member of the TestUtils class.